webapp/channels/src/components/pricing_modal/content.tsx

```diff
@@ -118,8 +121,20 @@ function Content(props: ContentProps) {
         trial_notification: isPreTrial,
     });
 
+    const getAdminProfessionalBtnText = () => {
+        if (currentSubscriptionIsMonthlyProfessional) {
+            return formatMessage({id: 'pricing_modal.btn.switch_to_annual', defaultMessage: 'Switch to annual billing'});
+        }
+
+        if (enabled) {
```

Comment:
Could we provide a more meaningful name to this as it's not clear what is enabled? Also, wondering if it should be the name should be reverse given that was is enable is the deprecation of cloud. So maybe:
```suggestion
        if (cloudFreeDeprecated) {
```